README.md

```diff
@@ -1 +1,102 @@
-# Global-Leadership-Summit
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+  - [🚀 Live Demo](#live-demo)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👤 Author](#author)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Global Leadership Summit] <a name="about-project"></a>
+
+> This project was created to design a website for an upcoming event which assume to be hosted on United States.
+>Its is Microverse Module 1 Capstone Project to test or skills on all that we have built in the past.
+> It is built with flexbox, css grid,  media query for responsive web design and JavaScript Dom manipulation for menubar.
+>It have two pages, the Home page and the about page
+
+**Global Leadership Summit]** is a webpage with menu bar, hero section, featured speakers section, about section and About Page.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> The tech stack used are html, css and js.
+
+
+<!-- LIVE DEMO -->
+
+## 🚀 Video Presentation link <a name="Video Presentation link"></a>
+- [Live Demo Link](https://www.loom.com/share/dcd06582bc6540608b4ae446cab6718d/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+### Deployment
+
+You can deploy this project using:
+- Github Page
+- Any public server.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHOR -->
+
+## 👤 Author1 <a name="Stephen"></a>
+
+
+- GitHub: [@githubhandle](https://github.com/chibunduonyeje1)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/chibunduonyeje1/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Image Slider]**
+- [ ] **[ChatBot]**
+- [ ] **[Testimonies Section]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/chibunduonyeje1/Global-Leadership-Summit/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
```

Comment:
Great job @chibunduonyeje1 🤝

- [ ] As stated in the project requirements, kindly make sure the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).